tests/integration/targets/ios_logging_global/vars/main.yaml

```diff
@@ -0,0 +1,147 @@
+---
+merged:
+  before: {}
+
+  commands:
+    - logging on
+    - logging policy-firewall rate-limit 10
+    - logging host 172.16.1.13 transport tcp port 514
+    - logging userinfo
+    - logging facility local5
+    - logging message-counter log
+    - logging dmvpn rate-limit 10
+    - logging host 172.16.1.12
+    - logging host 172.16.1.11 xml
+    - logging message-counter syslog
+    - logging buffered xml 5099 notifications
+    - logging console xml critical
+    - logging monitor warnings
+    - logging exception 4099
+    - logging host 172.16.1.10 filtered stream 10
+    - logging cns-events warnings
+    - logging trap errors
+
+  after:
```

Comment:
after dict maintained but it's not being compared with integration tests run, ideally before and after should be compared for all merge, replace and override state.